src/musicbrainz/musicbrainzclient.cpp

```diff
@@ -37,6 +39,9 @@ void MusicBrainzClient::start(int id, const QString& mbid) {
     url.setQueryItems(parameters);
     qDebug() << "MusicBrainzClient GET request:" << url.toString();
     QNetworkRequest req(url);
+    // http://musicbrainz.org/doc/XML_Web_Service/Rate_Limiting#Provide_meaningful_User-Agent_strings
+    QString mixxxMusicBrainzId(Version::applicationName() + "/" + Version::version() + " ( " + MIXXX_WEBSITE_URL + " )");
```

Comment:
Parenthesis are into the string given to MusicBrainz and they ask for spaces in their suggestion.
http://musicbrainz.org/doc/XML_Web_Service/Rate_Limiting#Provide_meaningful_User-Agent_strings